plugins/scaffolder-backend/package.json

```diff
@@ -34,7 +36,7 @@
     "express": "^4.17.1",
     "express-promise-router": "^3.0.3",
     "fs-extra": "^9.0.0",
-    "git-url-parse": "^11.1.2",
+    "git-url-parse": "^11.2.0",
```

Comment:
This version bump is required because the library didn't handle GitLab URLs with scope correctly (see IonicaBizau/git-url-parse#107).